yarn.lock

```diff
@@ -3240,7 +3240,12 @@ globjoin@^0.1.4:
   resolved "https://registry.yarnpkg.com/globjoin/-/globjoin-0.1.4.tgz#2f4494ac8919e3767c5cbb691e9f463324285d43"
   integrity sha512-xYfnw62CKG8nLkZBfWbhWwDw02CHty86jfPcc2cr3ZfeuK9ysoVPPEUxf21bAD/rWAgk52SuBrLJlefNy8mvFg==
 
-govuk-frontend@4.7.0, govuk-frontend@^4.5.0:
+govuk-frontend@5.0.0:
+  version "5.0.0"
+  resolved "https://registry.yarnpkg.com/govuk-frontend/-/govuk-frontend-5.0.0.tgz#c08a4d1115fb31eb39b6d19979c627f816185dd7"
+  integrity sha512-3WSfvQ+3kw/q/m8jrq/t8XnMUA8D2r0uhGyZaDbIh1gWTJBQzJBHbHiKYI9nc9ixIXdCFsc9RozkgEm57a795g==
+
+govuk-frontend@^4.5.0:
```

Comment:
why do we still need this?

Reply:
This module exists because "@x-govuk#govuk-prototype-components#govuk-prototype-kit" depends on it
Which I guess makes sense to update as well, so will do that now..

Reply:
Updated it now, though it looks like this dependency is still there